### PR TITLE
Add register stream provider F# helper

### PR DESCRIPTION
**Pull request by aprooks**
#113 

I've kept both versions of config setup, will make it more consistent after deciding which version to pick.

Atm I lean towards verbose but explicit version like in cluster config.

### CODE REVIEW

File: Source/Orleankka.FSharp/Configuration.fs [hunk 1]
```diff
@@ -1,61 +1,64 @@
...
-
-   let inline actorOfPath(system:IActorSystem) path =
-      system.ActorOf(path) |> FSharp.ActorRef<obj>
-
-   let inline typedActorOf<'TActor when 'TActor :> IActor> (system:IActorSystem, actorId) =
-      let actorPath = typeof<'TActor>.ToActorPath(actorId) 
-      system.ActorOf(actorPath) |> FSharp.ActorRef<'TActor>
-
-   let inline streamOf (system:IActorSystem) provider streamId = 
+﻿namespace Orleankka.FSharp.Configuration
```
Comment (aprooks) [inline Source/Orleankka.FSharp/Configuration.fs]: line endings :(

File: Source/Orleankka.FSharp/Task.fs [hunk 2]
```diff
@@ -1,230 +1,230 @@
-﻿namespace Orleankka.FSharp
```
Comment (aprooks) [inline Source/Orleankka.FSharp/Task.fs]: caused by line endings unification

**Comment by aprooks** [comment]
last time I converted only files I changed. `Task.fs` was a surprise

**Comment by yevhen** [comment]
@aprooks I believe that Task.fs was initially imported with wrong line endings. So, nevermind.

**Pull request merged**